config/remote/firebase.go

```diff
@@ -66,27 +67,16 @@ func (rc *RConfig) fetchRemoteConfigIfTime() error {
 		return fmt.Errorf("loading config: %w", err)
 	}
 
+	// don't fetch the remote config too often even if there is nothing cached
```

Comment:
Is this comment correct? `RCLastUpdate` is only updated after a successful fetch